app/templates/wizard/form.html

```diff
@@ -12,7 +12,7 @@
           <form class="form-horizontal col-md-8" role="form" action="{{url_for('.wizard_form', router_id = "/".join(router.id)) }}" method="post">
 
             {{ form.csrf_token }}
-            {{ render_bootstrap_field(form.hostname, "Spitzname des Routers", "Ein kurzer und eindeutiger Name für deinen Router (z.B. <em>funkenpumpe</em> oder <em>HermannStr42_VH</em>). Dieser Name wird nur intern für dich hinterlegt.") }}
+            {{ render_bootstrap_field(form.hostname, "Spitzname des Routers", "Ein kurzer und eindeutiger Name für deinen Router (z.B. <em>funkenpumpe</em> oder <em>HermannStr42_VH</em>). Dieser Name wird nur intern für dich hinterlegt und muss zwischen 4 und 32 Zeichen lang sein.") }}
```

Comment:
Good Idea might be to add, that this name should match the name, which gets typed into the firmware-wizard. That was a thing that @noki mentioned some time ago. Once we got the new firmware-wizard, we can set more strong incentives for this via the config-json-upload.